docs/tasks/debug-application-cluster/run-command-container-kubectl-exec.md

```diff
@@ -0,0 +1,131 @@
+---
+assignees:
+- caesarxuchao
+- mikedanese
+title: Running Commands in a Container using kubectl exec
+---
+
+{% capture overview %}
+
+This page shows how to use `kubectl exec` to run commands in a Container.
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+
+{% capture steps %}
+
+## Using kubectl exec to view the environment variables of a Container
+
+Kubernetes exposes
+[services](/docs/user-guide/services/#environment-variables) through environment variables.
+It is convenient to check these environment variables using `kubectl exec`.
+
+In this exercise, you create a Service and a Deployment. Here is the
+configuration file for the Service:
+
+{% include code.html language="yaml" file="redis-service.yaml" ghlink="/docs/tasks/debug-application-cluster/redis-service.yaml" %}
+
+In the configuration file, you can see that the Service provides access to
+Pods that have the the label `pod-is-for: kubectl-exec-demo`.
+
+Create the Service:
+
+```shell
+kubectl create -f https://k8s.io/docs/tasks/debug-application-cluster/redis-service.yaml
+```
+
+The next step is to create a Deployment. The Deployment has a one Pod, and the
+Pod has one Container that runs the `redis` image. Here is the configuration
+file for the Deployment:
+
+{% include code.html language="yaml" file="redis-deployment.yaml" ghlink="/docs/tasks/debug-application-cluster/redis-deployment.yaml" %}
+
+In the configuration file, you can see that the Pod has the label `pod-is-for: kubectl-exec-demo`.
```

Comment:
Drop all the stuff about labels for the first part of the task.  Just keep it simple - Create a Deployment, exec something in it.

Cover 2 pieces:

1. exec to get a shell with -i -t so the tty is setup
2. exec to run a command without a tty